components/security/authentication.rst

```diff
@@ -317,7 +317,7 @@ The ``security.switch_user`` event is triggered every time you activate
 the ``switch_user`` firewall listener.
 
 The ``Symfony\Component\Security\Http\Event\DeauthenticatedEvent`` event is triggered when a token has been deauthenticated
-because of a user change, it can help you doing some clean-up task when a logout has been triggered.
+because of a user change, it can help you doing some clean-up task when a forced logout has been triggered due to the user change.
```

Comment:
```suggestion
because of a user change, it can help you doing some clean-up task.
```

I think there is no need to duplicate the "when" from the first part of the sentence.

Reply:
👍🏻 I'm changing it.

Reply:
Done